eoscostarica/guias.eoscostarica.io
# Feat/pair programming

### GH Issue

Resolves #259 

### Steps to test
1. Check the Content
2. Go to enlaces útiles
3. Go to Contratos Inteligentes
4. Review sidebar


#### CheckList
